modules/Kconfig.cmsis

```diff
@@ -3,15 +3,19 @@
 
 config HAS_CMSIS_CORE
 	bool
-	select HAS_CMSIS_CORE_M if CPU_CORTEX_M
+	select HAS_CMSIS_CORE_A if CPU_CORTEX_A
 	select HAS_CMSIS_CORE_R if CPU_CORTEX_R
+	select HAS_CMSIS_CORE_M if CPU_CORTEX_M
 
 if HAS_CMSIS_CORE
 
-config HAS_CMSIS_CORE_M
+config HAS_CMSIS_CORE_A
 	bool
 
 config HAS_CMSIS_CORE_R
 	bool
 
+config HAS_CMSIS_CORE_M
```

Comment:
any particular reason A, M, R are not sorted alphabetically?

Reply:
Because the current order spells A R M? :D (always thought this was something nice to look at; also, it is not like we will have another CMSIS-Core variant in the foreseeable future).

If you insist that we go with the alphabetical order though, I will change it.

Reply:
No, that's fine 👍 

Actually, _this_ is the way they are listed in the ARM Infocenter, so let's keep it.